frontend/src/views/pages/inventory/Inventory.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div>
+    <CRow>
+      <CCol>
+        <CAlert show color="success" v-if="successMessage" class="mb-2">
+          {{ successMessage }}
+        </CAlert>
+        <CRow class="m-0 mb-3 d-flex justify-content-between">
+          <CButton color="primary" id="add-new-vehicle">
+            Add vehicle(s) to the inventory
+          </CButton>
+          <CButton
+            color="secondary"
+            id="set-dealership-default"
+            v-if="
+              !!selectedDealership &&
+              (!$store.state.auth.dealership ||
+                $store.state.auth.dealership != selectedDealership)
+            "
+            @click="setDefaultDealership"
+          >
+            Set selected dealership as default
+          </CButton>
+        </CRow>
+        <CSelect
+          v-if="$store.state.auth.role == 'Administration'"
+          id="dealership-select-cmb"
+          :options="adminDealerships"
+          :value.sync="selectedDealership"
+          placeholder="Select a dealership to view their inventory"
+          @change="selectDealership"
+        />
+        <CCard v-if="tableFields.length > 0">
+          <CCardHeader>
+            <slot name="header">Inventory list of vehicles</slot>
+          </CCardHeader>
+          <CCardBody>
+            <CDataTable
+              :fields="tableFields"
+              :items="tableItems"
+              :striped="true"
+              :items-per-page="10"
+              :fixed="true"
+              :clickable-rows="true"
+              @row-clicked="clickRow"
+            >
+            </CDataTable>
+          </CCardBody>
+        </CCard>
+      </CCol>
+    </CRow>
+  </div>
+</template>
+
+<script>
+const axios = require("axios");
+
+export default {
+  name: "Inventory",
+  data() {
+    return {
+      tableItems: [],
+      tableFields: [],
+      adminDealerships: [],
+      selectedDealership: null,
+      defaultAdminDealership: null,
+      successMessage: null,
+    };
+  },
+  methods: {
+    clickRow(row) {
+      console.log(row);
+    },
+    selectDealership() {
+      this.fetchVehicleProperties(this.selectedDealership);
+    },
+    showSuccessMessage(message) {
+      this.successMessage = message;
+      setTimeout(() => {
+        this.successMessage = null;
+      }, 5000);
+    },
+    setDefaultDealership() {
+      axios({
+        method: "PUT",
+        url: `${this.$store.state.api}/users/${this.$store.state.auth.userId}`,
+        headers: {
+          Authorization: `Bearer ${this.$store.state.auth.token}`,
+        },
+        data: {
+          dealership: this.selectedDealership,
+        },
+      })
+        .then((response) => {
+          if (response.data.success) {
+            //console.log(response.data.payload);
```

Comment:
Remove commented out code

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/37/commits/8c83bea65e9c8a6186a069f6415340392a55a7b4)